pkg/distributor/forwarding/forwarding.go

```diff
@@ -339,7 +341,8 @@ func (f *forwarder) splitToIngestedAndForwardedTimeseries(tsSliceIn []mimirpb.Pr
 			if filteredSamples > 0 {
 				err = errSamplesTooOld
 				if !ingest {
-					f.discardedSamplesTooOld.WithLabelValues(user).Add(float64(filteredSamples))
+					group := validation.FindGroupLabel(f.limits, user, ts.Labels)
```

Comment:
This extracts the group from each discarded timeseries rather than from the push request, can we make that consistent?